sections/main-cart-items.liquid

```diff
@@ -64,6 +64,27 @@
 
                     <a href="{{ item.url }}" class="cart-item__name h4 break">{{ item.product.title | escape }}</a>
 
+                    {%- if item.original_price != item.final_price -%}
+                      <div class="cart-item__discounted-prices">
+                        <span class="visually-hidden">
+                          {{ 'products.product.price.regular_price' | t }}
+                        </span>
+                        <s class="cart-item__old-price product-option">
+                          {{- item.original_price | money -}}
+                        </s>
+                        <span class="visually-hidden">
+                          {{ 'products.product.price.sale_price' | t }}
+                        </span>
+                        <strong class="cart-item__final-price product-option">
+                          {{ item.final_price | money }}
+                        </strong>
+                      </div>
+                    {%- else -%}
+                      <div class="product-option">
+                        {{ item.original_price | money }}
```

Comment:
I wonder if we should show the compare at price as well ? 🤔 And how it would work when there is both a discount and a compare at price.
Show the discount and if there is no discount applied then check if there is a compare at price on line 84.

Reply:
@ludoboludo Oh I see what you mean 🤔 I created an issue and tagged you and Meli: https://github.com/Shopify/dawn/issues/854

Feel free to add any comments

Reply:
> And how it would work when there is both a discount and a compare at price.

For that specific reason, we are not showing the original price on cart to avoid confusion. We don't show the original price either at Checkout. I'll try to find more docs related to that topic.